conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosNodeOperator.scala

```diff
@@ -21,6 +21,29 @@ private[tezos] object TezosNodeOperator {
   type LazyPages[T] = Iterator[Future[T]]
   type Paginated[T, N] = (LazyPages[T], N)
 
+  /** Process every single page of results by converting the content with a custom function.
+    * The returned value will be the content of the paginated results.
+    */
+  def mapByPage[T](pages: LazyPages[T])(pageMapper: T => T)(implicit ec: ExecutionContext): LazyPages[T] =
```

Comment:
Doesn't look like this function is being used anywhere. Is this left here for future use?

Reply:
Indeed, to me it looked like a missing piece and without it the other felt very ad-hoc... but in the end we could remove it.

Actually I'd need some time to check if we could put it to use in some place already, but possibly in a separate task work. I would make a separate PR to use it or remove it, based on the outcome of a quick survey